app/ante/min_fee_test.go

```diff
@@ -0,0 +1,86 @@
+package ante_test
```

Comment:
`fee_checker_test.go` already exists. I had to split up the test suit because of circular dependency issues since `getTxPriority` is not exported  and has to be part of the same package as `fee_checker.go`, nevertheless I think you're right this is confusing. I'll rename `fee_checker_test.go` `to tx_priority_test.go` and leave this file as is.